aikau/src/main/resources/alfresco/menus/AlfMenuBarSelectItems.js

```diff
@@ -176,19 +176,22 @@ define(["dojo/_base/declare",
        * @param {object} payload
        */
       determineSelection: function alfresco_menus_AlfMenuBarSelectItems__determineSelection(payload) {
-         if (payload.availableItemCount != null && payload.selectedItemCount != null) 
+         var available = parseInt(payload.availableItemCount, 10);
```

Comment:
if payload.availableItemCount or payload.selectedItemCount will be null or undefined NaN will be returned.